### ISSUE TITLE
Progress Bar (Or some indication that it isn't stuck)

### ISSUE THREAD

**Issue by mpchadwick** [rhoerr/supee-6788-toolbox#4]
When I ran this there were nine minutes in between

"---- Searching files for bad routes -------------------------------" and the next line of output (checked the timestamps in the log to get that number)

https://github.com/rhoerr/supee-6788-toolbox/blob/186ecb2cada4913c9e8850635c15a6bb780518ce/fixSUPEE6788.php#L100

I nearly killed it because I thought it was stuck. Could be nice to show a progress bar, or even just periodically update the user that the script isn't stuck.

**Comment by rhoerr** [comment]
Nine minutes is a long runtime. Most of the ones I've seen have been <4 minutes.

The major slow part is this chunk of code: `if( preg_match( '/addFieldToFilter\(\s*[\'"]?[\`(]/i', $line ) ) {` -Removing that would save you about 80% of execution time, but means no scan at all for the SQL change.

It wouldn't be too hard to add some sort of message every thousand files or such. I didn't want to clutter up the output.

**Comment by mpchadwick** [comment]
Yeah, I agree about cluttering the output, maybe there is a "verbose" flag when executing, which is documented in the README, but off by default? The 9 minutes was in my dev VM which isn't tremendously powerful. When I ran it on a prod box it was much quicker.
